osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -25,6 +25,19 @@ public class OsuPerformanceCalculator : PerformanceCalculator
         private int countMeh;
         private int countMiss;
 
+        /// <summary>
+        /// Missed slider ticks that includes missed reverse arrows. Will only be correct on non-classic scores
+        /// </summary>
+        private int countSliderTickMiss;
+
+        /// <summary>
+        /// Amount of missed slider tails that don't break combo. Will only be correct on non-classic scores
+        /// </summary>
+        private int countSliderEndsDropped;
```

Comment:
Since slider-related code uses the term "tail" (and the hit result does too), I'd suggest renaming this to `countSliderTailMiss` or `countSliderTailsDropped`. I'd prefer to consider "miss" the opposite of hit even for slider tails but calling them "dropped" is arguable